weblate/trans/models/subproject.py

```diff
@@ -786,9 +786,10 @@ def do_push(self, request, force_commit=True, do_update=True):
 
         # Do actual push
         try:
+            fork_branch = '{0}-weblate'.format(self.name)
```

Comment:
This really should live in the GitHub VCS backend, there is no need to put this to generic code.